Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -21,6 +21,7 @@
 #include "ProgramStatus.h"
 #include <string.h>
 
+
```

Comment:
Get rid of the commit that causes the file changes here, since nothing changed in these files.